src/rsem_quant.py

```diff
@@ -77,6 +93,14 @@ def main(args):
             bam_root=bam_root))
     logger.info('Running RSEM command %s', ' '.join(rsem_call))
     subprocess.call(rsem_call)
+    gene_quant_fn = str(bam_root) + '_rsem.genes.results'
+    number_of_genes_detected = calculate_number_of_genes_detected(
+        gene_quant_fn)
+    number_of_genes_detected_dict = {
+        'number_of_genes_detected': number_of_genes_detected
+    }
+    with open(str(bam_root) + '_number_of_genes_detected.json', 'w') as f:
+        json.dump(number_of_genes_detected_dict, f)
```

Comment:
Test coverage is through workflow tests, right?

Reply:
Correct. I will change this so that it uses QCMetric and QCMetricRecord from rna_qc, now that I have those.